docs/tracking.md

```diff
@@ -113,3 +113,5 @@ please open an issue on [this issue tracker](https://github.com/Quansight-Labs/f
 [^1]: Release available in the [Bazel Central Registry](https://registry.bazel.build)
 
 [^2]: Rust library released on [crates.io](https://crates.io/crates/pyo3)
+
+[^3]: Planned release.
```

Comment:
Can you please remove this and the `[^3]` higher up? This is already covered by the lack of a checkmark in the "PyPI release column", so this footnote doesn't add anything except extra work to remove it again later. We can always fill in release numbers when we know them.